src/components/common/ui/form/numbered-text-area-mui/NumberedTextArea.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const wrapper: SxProps<Theme> = {
+  width: '100%',
+};
+export const formControl = (
+  state: string,
+  disabled: boolean,
+): SxProps<Theme> => ({
+  width: '100%',
+  position: 'relative',
+  display: 'flex',
+  flexDirection: 'row',
+  paddingBottom: '7px',
+  border: '2px solid',
+  borderRadius: '8px',
+  transition: 'all 0.2s ease-in-out',
+
+  ...(state === 'default' &&
+    !disabled && {
+      borderColor: 'grey.500',
+      color: 'grey.500',
+      '&:hover': {
+        color: 'grey.700',
+        borderColor: 'grey.700',
+      },
+      '&:focus-within': {
+        color: 'grey.700',
+        borderColor: 'grey.700',
+      },
+    }),
+
+  ...(state === 'error' &&
+    !disabled && {
+      borderColor: 'error.500',
+    }),
+
+  ...(disabled && {
+    color: 'grey.400',
+    borderColor: 'grey.400',
+  }),
```

Comment:
can we rewrite this as
 
  ...(disabled && {
    color: 'grey.400',
    borderColor: 'grey.400',
  }),

  ...(state === 'default' && {
    borderColor: 'grey.500',
    color: 'grey.500',
    '&:hover': {
      color: 'grey.700',
      borderColor: 'grey.700',
    },
    '&:focus-within': {
      color: 'grey.700',
      borderColor: 'grey.700',
    },
  }),

  ...(state === 'error' && {
    borderColor: 'error.500',
  }),